index.html

```diff
@@ -8,5 +8,7 @@
 </head>
 <body>
     <h1>Git Demo</h1>
+    <img src="https://media.giphy.com/media/CWkoXi05ji0Ny/giphy.gif" alt="Unlimited Power">
```

Comment:
alt should have the text "this is a nice image"

Reply:
got it!